app/models/project.rb

```diff
@@ -0,0 +1,100 @@
+# == Schema Information
+#
+# Table name: projects
+#
+#  id              :integer          not null, primary key
+#  name            :string(255)
+#  model_id        :integer
+#  origin_id       :integer
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  target_size     :integer
+#  current_size    :integer
+#  comments        :text
+#  admin_user_id   :integer
+#  project_type_id :integer
+#
+
+class Project < ActiveRecord::Base
+    include Elasticsearch::Model
+
+  belongs_to :country, foreign_key: 'origin_id'
+  has_many :content_buckets
+  has_and_belongs_to_many :languages
+  has_and_belongs_to_many :levels
+  has_and_belongs_to_many :users, association_foreign_key: 'admin_user_id'
+
+  settings number_of_shards: 1 do
+    mapping do
+      indexes :country_name, index: 'not_analyzed'
+      indexes :languages_names, index: 'not_analyzed'
+      indexes :levels_names, index: 'not_analyzed'
+    end
+  end
+
+  def as_indexed_json(options={})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `as_indexed_json(*)` if you want the method to accept any arguments but don't care about them.<br>Inconsistent indentation detected.<br>Surrounding space missing in default value assignment.